plugins/micromega/coq_micromega.ml

```diff
@@ -731,7 +731,7 @@ struct
     (** [eq_constr gl x y] returns an updated [gl] if x and y can be unified *)
     let eq_constr gl x y =
       let evd = gl.sigma in
-      match EConstr.eq_constr_universes gl.env evd x y with
+      match EConstr.eq_constr_universes_proj gl.env evd x y with
```

Comment:
This change isn't mentioned by the PR title.

Reply:
Updated.